www/c-icap/src/opnsense/service/templates/OPNsense/CICAP/virus_scan.conf

```diff
@@ -0,0 +1,25 @@
+{% if helpers.exists('OPNsense.cicap.antivirus.enable_clamav') and OPNsense.cicap.antivirus.enable_clamav == '1' %}
+Service antivirus_module virus_scan.so
+ServiceAlias srv_clamav virus_scan
+ServiceAlias  avscan virus_scan?allow204=on&sizelimit=off&mode=simple
+virus_scan.ScanFileTypes TEXT DATA EXECUTABLE ARCHIVE GIF JPEG MSOFFICE
```

Comment:
@fabianfrz Found the typo. But now conf is generated like this:
virus_scan.ScanFileTypes ['TEXT,DATA,GIF']